Action fails with dotnet version 7.x.x

When using `actions/setup-dotnet` with version `7.x.x`, the job consistently fails with the error message:
```
Can't find suitable dotnet or node installation to install retype package with.
```

Workflow file:
```
    steps:
      - name: Checkout repo
        uses: actions/checkout@v3
        with:
          fetch-depth: 0

      - uses: actions/setup-dotnet@v2
        with:
          dotnet-version: 7.0.x

      - name: Build doc site
        uses: retypeapp/action-build@latest
        with:
          license: ${{ secrets.RETYPE_LICENSE_KEY }}
```

I believe this line https://github.com/retypeapp/action-build/blob/main/build.sh#L28 is only checking for dotnet --version == 5 which is causing this issue. I tried to use setup-dotnet with dotnet-version: 5.x.x as a workaround, but this didn't work for retype version 3.0.3 as it required atleast dotnet v 7.

I tried the fix you've provided `retypeapp/action-build@issue-1648` and that resolved our initial issue! However, we're now seeing a different error:

```
Run retypeapp/action-build@issue-1648
Run ${GITHUB_ACTION_PATH}/build.sh
Working directory is: /runner/_work/container-infra/container-infra
Installing Retype v3.0.3 using dotnet tool: done.
Determining temporary target folder to place parsed documentation: /tmp/tmp.7LDcJCN6Lf
Setting up build arguments: /retype.yml, license key, done.
Building documentation: error.
Error: retype build command failed with exit code 1.
Failed command and output:

$ retype build --verbose --secret *** --override {
  "output": "/tmp/tmp.7LDcJCN6Lf"
}

/runner/_work/_actions/retypeapp/action-build/issue-1648/build.sh: line 158: retype: command not found
----

Aborting documentation build process.
Error: Process completed with exit code 1.
```

I think this could be resolved by updating the path in the build script?
```
export PATH="$PATH:/home/runner/.dotnet/tools"
```

--- a/.github/workflows/refresh-release.yml
+++ b/.github/workflows/refresh-release.yml
@@ -6,7 +6,7 @@ jobs:
     name: Sync Retype and Release
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Update and Release
         shell: bash
         run: .github/workflows/refresh-release.sh --github-token="${{ secrets.GITHUB_TOKEN }}"

--- a/README.md
+++ b/README.md
@@ -14,7 +14,7 @@ This action will look for a [`retype.yml`](https://retype.com/configuration/proj
 
 ```yaml
 steps:
-- uses: actions/checkout@v2
+- uses: actions/checkout@v3
 
 - uses: retypeapp/action-build@latest
 ```
@@ -27,7 +27,7 @@ The workflow file above would then become:
 
 ```yaml
 steps:
-- uses: actions/checkout@v2
+- uses: actions/checkout@v3
 
 - uses: actions/setup-dotnet@v1
   with:
@@ -71,7 +71,7 @@ jobs:
       contents: write
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - uses: retypeapp/action-build@latest
 ```
@@ -126,11 +126,11 @@ The config file may have a different file name
 In a bit more complex scenario where various repositories are checked out in a workflow. This may be useful, for instance, if retype documentation is generated from files across different repositories.
 
 ```yaml
-- uses: actions/checkout@v2
+- uses: actions/checkout@v3
   with:
     path: own-repository
 
-- uses: actions/checkout@v2
+- uses: actions/checkout@v3
   with:
     repository: organization/repository-name
     path: auxiliary-repository

--- a/README.md
+++ b/README.md
@@ -182,8 +182,16 @@ The following sample demonstrates configuring the Retype `action-github-pages` a
 ```yaml
 - uses: retypeapp/action-build@latest
 
-- uses: retypeapp/action-github-pages@v2
+- uses: retypeapp/action-github-pages@latest
   with:
     branch: retype
     update-branch: true
+```
+
+## Testing with a specific branch of the retypepp action
+
+You can test with a specific branch of the retypapp action by replacing the `@latest` with the `@branch-name-here`.
+
+```yaml
+- uses: retypeapp/action-build@branch-name-here
 ```

--- a/build.sh
+++ b/build.sh
@@ -25,7 +25,7 @@ fi
 echo "Working directory is: $(pwd)"
 
 # We prefer dotnet if available as the package size is (much) smaller.
-if which dotnet > /dev/null 2>&1 && [ "$(dotnet --version | cut -f1 -d.)" == "5" ]; then
+if which dotnet > /dev/null 2>&1 && [ "$(dotnet --version | cut -f1 -d.)" -ge 5 ]; then
   use_dotnet=true
 elif ! which node > /dev/null 2>&1 || [ "$(node --version | cut -f1 -d. | cut -b2-)" -lt 14 ]; then
   fail "Can't find suitable dotnet or node installation to install retype package with."

--- a/build.sh
+++ b/build.sh
@@ -28,7 +28,7 @@ echo "Working directory is: $(pwd)"
 if which dotnet > /dev/null 2>&1 && [ "$(dotnet --version | cut -f1 -d.)" -ge 5 ]; then
   use_dotnet=true
 elif ! which node > /dev/null 2>&1 || [ "$(node --version | cut -f1 -d. | cut -b2-)" -lt 14 ]; then
-  fail "Can't find suitable dotnet or node installation to install retype package with."
+  fail "Cannot find a suitable dotnet or node installation to install the retype package with."
 fi
 
 retype_path="$(which retype 2> /dev/null)"

